src/app/routes/app-routes.tsx

```diff
@@ -83,151 +202,40 @@ function useAppRoutes() {
       )
     );
 
-  const settingsModalRoutes = (
-    <Route>
-      <Route path={RouteUrls.SignOutConfirm} element={<SignOutConfirmDrawer />} />
-      <Route path={RouteUrls.ChangeTheme} element={<ThemesDrawer />} />
-      <Route path={RouteUrls.SelectNetwork} element={<SelectNetwork />} />
-    </Route>
-  );
-
-  const legacyRequestRoutes = (
-    <>
-      <Route
-        path={RouteUrls.TransactionRequest}
-        element={
-          <AccountGate>
-            <Suspense fallback={<LoadingSpinner height="600px" />}>
-              <TransactionRequest />
-            </Suspense>
-          </AccountGate>
-        }
-      >
-        {ledgerStacksTxSigningRoutes}
-        <Route path={RouteUrls.EditNonce} element={<EditNonceDrawer />} />
-        <Route path={RouteUrls.TransactionBroadcastError} element={<BroadcastErrorDrawer />} />
-      </Route>
-      <Route
-        path={RouteUrls.SignatureRequest}
-        element={
-          <AccountGate>
-            <Suspense fallback={<LoadingSpinner height="600px" />}>
-              <StacksMessageSigningRequest />
-            </Suspense>
-          </AccountGate>
-        }
-      >
-        {ledgerStacksMessageSigningRoutes}
-      </Route>
-      <Route
-        path={RouteUrls.ProfileUpdateRequest}
-        element={
-          <AccountGate>
-            <Suspense fallback={<LoadingSpinner height="600px" />}>
-              <ProfileUpdateRequest />
-            </Suspense>
-          </AccountGate>
-        }
-      />
-      <Route
-        path={RouteUrls.PsbtRequest}
-        element={
-          <AccountGate>
-            <Suspense fallback={<LoadingSpinner height="600px" />}>
-              <PsbtRequest />
-            </Suspense>
-          </AccountGate>
-        }
-      />
-    </>
-  );
-
-  const rpcRequestRoutes = (
-    <>
-      <Route
-        path={RouteUrls.RpcGetAddresses}
-        element={
-          <AccountGate>
-            <RpcGetAddresses />
-          </AccountGate>
-        }
-      />
-      {rpcSendTransferRoutes}
-      <Route
-        path={RouteUrls.RpcSignBip322Message}
-        lazy={async () => {
-          const { RpcSignBip322MessageRoute } = await import(
-            '@app/pages/rpc-sign-bip322-message/rpc-sign-bip322-message'
-          );
-          return { Component: RpcSignBip322MessageRoute };
-        }}
-      />
-      <Route
-        path={RouteUrls.RpcSignPsbt}
-        element={
-          <AccountGate>
-            <RpcSignPsbt />
-          </AccountGate>
-        }
-      />
-    </>
-  );
-
   return createHashRouter(
     createRoutesFromElements(
```

Comment:
I need to remove the path `/` from the root container and apply `*` to Home routes to prevent the warning about invalid nesting